lib/onebox/engine/github_actions_onebox.rb

```diff
@@ -63,20 +65,22 @@ def type
       end
 
       def data
+        result = raw(github_auth_header).clone
```

Comment:
Wondering if a shallow clone is enough, given this is a nested data structure. We might end up mutating nested arrays or hashes when we don't want to.